core/designsystem/build.gradle

```diff
@@ -37,4 +37,5 @@ android {
 dependencies {
     implementation libs.bundles.compose
     implementation libs.coil
+    implementation 'androidx.test.ext:junit-ktx:1.1.5'
```

Comment:
If you want to add dependencies, you have to follow the rules for adding new dependencies, consult with @irv205 or @dmroldanch